R/update-news.R

```diff
@@ -22,7 +22,7 @@ collect_news <- function(messages) {
     unlist()
 
   if (length(message_items) == 0) {
-    if (length(range) <= 1) {
+    if (length(messages) <= 1) {
```

Comment:
The logic seems repeated here. Should `no_change()` get a `messages` argument?

Reply:
I've changed the variable names to clarify that we are looking for `newsworthy_items` among `messages`.
Even if `no_change()` was run before getting to this point, if the argument `no_change_behavior` could mean we arrive at this point.
But maybe in the case there's less than or 1 message the function should exit earlier, is this what you mean?

Reply:
Should we have `no_change()` handle the `<= 1` comparison, so that this logic is centralized, if it makes sense?